src/PwdResetEmailSentController.js

```diff
@@ -40,6 +40,17 @@ function (Okta, Enums, FormController, FormType) {
       attributes: { 'data-se': 'pwd-reset-email-sent' },
       formChildren: function () {
         return [
+          FormType.View({
+            View: Okta.View.extend({
+              template: '\
+              <span class="overflow-hidden" style="width:0; height:0; display:block" role="alert">{{alert}}</span>\
```

Comment:
I'm not 100% sure what the ideal way to solve this problem is. My goal is to solve the immediate issue that our customer has raised. I'm hesitant to build a highly generic solution before we have more experience understanding these issues. That said, I can give it a try tonight and see how it works out.